_projects/use_coala_3.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - docker-coala-base
-desc: "Implement coala CI on a popular GitHub project, of your choice, using a Docker image on Travis CI, and report the results to the coala developers."
+desc: "Implement coala CI on a popular GitHub project, of your choice, using a Docker image on Travis CI, and report
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of preceeding line.